doc/requirements-doc.txt

```diff
@@ -1,6 +1,7 @@
 # Production requirements. This is what readthedocs.org picks up
 
 # Python / ML libraries
+accelerate>=0.17.0
```

Comment:
why do we need the actual accelerate library for the docs?

Reply:
This is because the python files are imported during doc build. If a library is missing, then the doc build fails. The alternative is to mock the module, but some modules like accelerate which do unorthodox stuff on import can still fail with that.